### DISCUSSION TITLE
ERROR: Cannot import Wazuh GPG key

### DISCUSSION THREAD

**Discussion by Tuxman2** [wazuh/wazuh#20145] [Q&A]
Hello,

I launched the Wazuh installation assistant script but it stops with the 'ERROR: Cannot import Wazuh GPG key' message. So, I can't install wazuh and all the components.

Can you help me ?

Thanks.

Wazuh version : 4.6
Ubuntu 22.04.x LTS server (x86_64)
wazuh-install.sh script

**Comment by rauldpm** [comment]
Hello

What is the output of executing the script using the `-v` option? Can you share the output of the file `/var/log/wazuh-install.log`?

If you run this step manually, do you have the same problem?

> curl -s https://packages.wazuh.com/key/GPG-KEY-WAZUH | gpg --no-default-keyring --keyring gnupg-ring:/usr/share/keyrings/wazuh.gpg --import && chmod 644 /usr/share/keyrings/wazuh.gpg

Please transfer the discussion to the `wazuh-packages` repository since that is where issues related to the installation assistant are discussed or created, if you cannot do it, tell me and I will transfer it
